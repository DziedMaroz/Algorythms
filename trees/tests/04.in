10
15
5
6
4
14
16
