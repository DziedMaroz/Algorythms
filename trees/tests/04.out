10
5
4
6
15
14
16
